core/templates/dev/head/pages/admin/admin.controller.ts

```diff
@@ -13,18 +13,10 @@
 // limitations under the License.
 
 /**
- * @fileoverview Data and controllers for the Oppia admin page.
+ * @fileoverview Controllers for the Oppia admin page.
  */
 
-oppia.constant('ADMIN_HANDLER_URL', '/adminhandler');
-oppia.constant('ADMIN_ROLE_HANDLER_URL', '/adminrolehandler');
-oppia.constant('PROFILE_URL_TEMPLATE', '/profile/<username>');
-oppia.constant(
-  'ADMIN_JOB_OUTPUT_URL_TEMPLATE', '/adminjoboutput?job_id=<jobId>');
-oppia.constant(
-  'ADMIN_TOPICS_CSV_DOWNLOAD_HANDLER_URL', '/admintopicscsvdownloadhandler');
-
-oppia.controller('Admin', [
+angular.module('adminModule').controller('Admin', [
```

Comment:
``` Admin ```  -> ``` admin```

Reply:
Ah I don't think that should be necessary, we are not changing any of the controller's names as of now so you can let it as it is.

Reply:
Yes, I agree with you @YashJipkate but shouldn't you think it should be done. Since we were refactoring the codebase and like other files, it should also be followed by the new naming convention.
What're your views @ankita240796?